docs/cli.rst

```diff
@@ -30,3 +30,29 @@ development and testing.
    :module: airflow.bin.cli
    :func: get_parser
    :prog: airflow
+
```

Comment:
I think there is a different kind of documentation in this file. We should not mix reference documentation and topic guide. This confuses readers. The same page should also provide information on how to configure the CLI to use the REST API

Reply:
Any suggestion on where we can add it?

Reply:
I am moving it to How-to guides

Reply:
Done

Reply:
LGTM. In the future I will try to organize the section related to CLI into one article.